UnityProject/Assets/Scripts/ScriptableObjects/ChemistryReagentsSO.cs

```diff
@@ -10,13 +11,13 @@ namespace ScriptableObjects
 	public class ChemistryReagentsSO : SingletonScriptableObject<ChemistryReagentsSO>
 	{
 		[SerializeField]
-		private Reagent[]  allChemistryReagents = new Reagent[0];
+		private List<Reagent> allChemistryReagents = new List<Reagent>();
```

Comment:
why change this to list?

Reply:
I think a little bit ago there was bugs with SOs and arrays but might be fixed now, doesn't really make any difference

Reply:
I thought lists were more expensive than arrrays

Reply:
https://stackoverflow.com/questions/454916/performance-of-arrays-vs-lists